web/src/enterprise/campaigns/detail/CampaignUpdateSelection.tsx

```diff
@@ -16,13 +16,21 @@ interface Props {
 /**
  * A list of a campaign's to choose from for an update
  */
-export const CampaignUpdateSelection: React.FunctionComponent<Props> = ({ history, location, onSelect }) => {
+export const CampaignUpdateSelection: React.FunctionComponent<Props> = ({ history, location }) => {
+    const patchSetID = useMemo(() => new URLSearchParams(location.search).get('patchSet'), [location.search])
+    const selectCampaign = useCallback<(campaign: Pick<GQL.ICampaign, 'id'>) => void>(
+        campaign => history.push(`/campaigns/${campaign.id}?patchSet=${patchSetID!}`),
```

Comment:
Suggestion: This is more readable imo
```suggestion
    const selectCampaign = useCallback(
        (campaign: Pick<GQL.ICampaign, 'id'>) => history.push(`/campaigns/${campaign.id}?patchSet=${patchSetID!}`),
```